js/shopping-list-item.js

```diff
@@ -0,0 +1,17 @@
+class ShoppingListItem {
+  constructor(name, description) {
```

Comment:
Your Class should do some amount of validation to make sure the user passes appropriate data. Assume both `name` and `description` are required, and throw an Error if either is not provided, or if they are not strings.